api/v1/cluster_types.go

```diff
@@ -665,24 +665,30 @@ type CertificatesStatus struct {
 	Expirations map[string]string `json:"expirations,omitempty"`
 }
 
-// BootstrapInitDB is the configuration of the bootstrap process when
-// initdb is used
-// Refer to the Bootstrap page of the documentation for more information.
-type BootstrapInitDB struct {
+// DatabaseInfo contains the information to connect to a user database
+type DatabaseInfo struct {
```

Comment:
Is this not compatible with old `initDB` section ? existed e2e need be refactored if we support new databaseInfo section